easybuild/easyconfigs/b/binutils/binutils-2.28-GCCcore-6.4.0.eb

```diff
@@ -1,21 +1,25 @@
+easyblock = 'EB_binutils'
```

Comment:
@JackPerdue These unneeded `easyblock` specs should be removed too please (both in `binutils` & `flex` easyconfigs in this PR)...

I understand there was confusion with EasyBuild v1.x since then we had a double-default (first look for software-specific easyblock, if that's not found fall back to `ConfigureMake`), but since EasyBuild v2.0 the fallback to `ConfigureMake` has been disabled (see http://easybuild.readthedocs.io/en/latest/Removed-functionality.html#depr-configuremake-fallback-eb1), so the situation is clear:

* if an `easyblock` line is present, use the specified easyblock
* else, use the software-specific easyblock, derived from `name` (and fail if it's not found)

So, there's no ambiguity anymore imho...